articles/vmware/vmw-how-access-centos-update-servers.md

```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
 
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories for CentOS 6 on our Elevated OFFICIAL security domain. To access them:
+We provide repositories for CentOS 6 and 7 on our Elevated OFFICIAL security domain. To access them:
 
 Create a file called `il3-repos.repo` in `/etc/yum.repos.d`, and populate it with the following:
 
```

Comment:
CentOS 6 looks like so:

[base]
name=CentOS-6-Base
baseurl=http://x.x.x.xcentos-base/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6

#released updates
[updates]
name=CentOS-6-Updates
baseurl=http://x.x.x.x/centos-updates/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6

#additional packages that may be useful
[extras]
name=CentOS-6-Extras
baseurl=http://x.x.x.x/centos-extras/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6